sample/src/main/AndroidManifest.xml

```diff
@@ -26,8 +26,8 @@
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="airbnb" />
-                <data android:scheme="http" />
+                <data android:scheme="dld" />
+                <data android:scheme="http" android:host="example.com" />
```

Comment:
renaming/updating this to avoid people from unadvertedly copy/pasting the sample code into their apps, which would cause weird problems by using the airbnb scheme and a catchall http intent filter